SSL3_GET_SERVER_CERTIFICATE:certificate verify failed

Hello The full error is : 



/Library/Python/2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/udemy-dl", line 5, in <module>
    main()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/udemy_dl/udemy_dl.py", line 194, in main
    udemy_dl(username, password, link, output_dir)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/udemy_dl/udemy_dl.py", line 141, in udemy_dl
    login(username, password)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/udemy_dl/udemy_dl.py", line 44, in login
    csrf_token = get_csrf_token()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/udemy_dl/udemy_dl.py", line 38, in get_csrf_token
    response = session.get('https://www.udemy.com/join/login-popup')
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/udemy_dl/udemy_dl.py", line 28, in get
    return self.session.get(url, headers=self.headers)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 477, in get
    return self.request('GET', url, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 465, in request
    resp = self.send(prep, **send_kwargs)
  File "/Library/Python/2.7/site-packages/requests/sessions.py", line 573, in send
    r = adapter.send(request, **kwargs)
  File "/Library/Python/2.7/site-packages/requests/adapters.py", line 431, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: [Errno 1] _ssl.c:499: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed

--- a/src/udemy_dl/download.py
+++ b/src/udemy_dl/download.py
@@ -21,11 +21,13 @@ def download(link, filename):
 
 
 def curl_dl(link, filename):
-    command = ['curl', '-C', '-', link, '-o', filename,'--insecure']
+    command = ['curl', '-C', '-', link, '-o', filename]
 
     cert_path = requests.certs.where()
     if cert_path:
         command.extend(['--cacert', cert_path])
+    else:
+        command.extend(['--insecure'])
     subprocess.call(command)
 
 

--- a/src/udemy_dl/udemy_dl.py
+++ b/src/udemy_dl/udemy_dl.py
@@ -2,6 +2,7 @@
 # -*- coding: utf8 -*-
 
 import requests
+import requests.sessions
 import argparse
 import getpass
 import sys
@@ -18,8 +19,8 @@ class Session:
                'Referer': '	http://www.udemy.com/'}
 
     def __init__(self):
-        self.session = requests.Session()
-
+        self.session = requests.sessions.Session()
+        
     def set_auth_headers(self, access_token, client_id):
         self.headers['X-Udemy-Bearer-Token'] = access_token
         self.headers['X-Udemy-Client-Id'] = client_id

